flutter/example/lib/main.dart

```diff
@@ -37,8 +37,12 @@ Future<void> main() async {
   }, (error, stackTrace) async {
     print('Capture from runZonedGuarded $error');
     final event = SentryEvent(
-      exception: error,
-      stackTrace: stackTrace,
+      exception: SentryException(
+        type: '${error.runtimeType}',
+        value: '$error',
+        stacktrace: SentryStackTrace()
+          ..frames = SentryStackTraceFactory().getStackFrames(stackTrace),
```

Comment:
I don't think this should be part of the sample, `SentryStackTraceFactory` and `SentryExceptionFactory` are implementation details.

`error` and `stackTrace` should be converted in the event pipeline (SentryClient), using the convert factories

Reply:
✅  removed